src/Type/Doctrine/Descriptors/EnumType.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type\Doctrine\Descriptors;
+
+use PHPStan\Type\StringType;
+use PHPStan\Type\Type;
+
+class EnumType implements DoctrineTypeDescriptor
```

Comment:
Am I wrong or without this type the data is considered as mixed ?

Now with this, it will report error `foo be 'A'|'B' but is string` on level 8 because the values option is not considered in the following code
```
#[ORM\Column(name: 'foo', type: Types::ENUM, options: ['values' => ['A', 'B'])]
```